src/widgets/unfold_widgets/algorithm_setup_widget.py

```diff
@@ -80,23 +80,41 @@ def __init__(self, parent, engine):
             )
         )
 
-        self.second_column.addWidget(self.options_group)
-        self.second_column.addStretch()
-
         # animation group
         self.animation_group = QGroupBox()
         self.animation_group.setTitle("Animation")
-        self.animation_group.setMinimumSize(220, 200)
-        self.animation_group_layout = QFormLayout(self.animation_group)
+        self.animation_group.setMinimumSize(220, 65)
+        self.animation_group_layout = QHBoxLayout(self.animation_group)
 
         self.animation_type = QComboBox()
         self.animation_type.addItems(["Step by step", "Animation", "No visualization"])
-        self.animation_group_layout.addRow(
-            QLabel("Visualization type"), self.animation_type
+        self.animation_type.setFixedWidth(175)
+
+        self.animation_group_layout.addWidget(QLabel("Visualization type"))
+        self.animation_group_layout.addStretch()
+        self.animation_group_layout.addWidget(self.animation_type)
+
+        self.second_column.addWidget(self.options_group)
+        self.second_column.addWidget(self.animation_group)
+
+        # description group
+        self.algorithm_description_group = QGroupBox()
+        self.algorithm_description_group.setTitle("Description")
+        self.algorithm_description_group.setFixedWidth(280)
+        self.algorithm_description_group.setMinimumHeight(265)
+        self.algorithm_description_group_layout = QVBoxLayout(
+            self.algorithm_description_group
         )
+        self.algorithm_description = QLabel()
+        self.algorithm_description.setText(
+            self.engine.get_algorithm_description(
+                self.technique_box.currentText(), self.algorithm_box.currentText()
+            )
+        )
+        self.algorithm_description.setWordWrap(True)
```

Comment:
```suggestion
        self.algorithm_description.setWordWrap(True)
        self.algorithm_description.setAlignment(Qt.AlignJustify)
```
i think it looks better justified